force-app/main/default/lwc/timelineItem/timelineItem.js

```diff
@@ -150,9 +165,18 @@ export default class TimelineItem extends NavigationMixin(LightningElement) {
     }
 
     get isOverride() {
-        return this.row.record.subtitleOverride != null ? (this.expanded === true ? false : true) : false;
+        return this.row.record.subtitleOverride != null;
+    }
+
+    get isOverrideAndNotExpanded() {
+        return this.isOverride && !this.expanded;
     }
+
     get isCustom() {
         return this.row.record.customComponent != null && this.row.record.customComponent != '';
     }
+
+    get clampOverride() {
+        if(this.row.record.clampLines != null) return '--lwc-lineClamp: ' + this.row.record.clampLines;
```

Comment:
forslag:    return this.row.record.clampLines != null ? `--lwc-lineClamp: ${this.row.record.clampLines}` : null eller undefined;